build.sbt

```diff
@@ -114,6 +128,16 @@ def module(name: String) = {
 }
 
 lazy val rlCore = module("core").settings(
+  initialCommands :=
+    """
+import io.samritchie.rl._
```

Comment:
makes exploration a little easier. I'll move these to the `book` project when I build that.